code/game/objects/structures/fireaxe.dm

```diff
@@ -69,9 +69,9 @@
 		update_icon()
 
 /obj/structure/fireaxecabinet/blob_act()
-	if(prob(75) && fireaxe)
```

Comment:
Notable stupid shit found in blob_act()
If there was no fireaxe in the fireaxe cabinet, it would not be deleted by the blob.